app/controllers/dependencies_controller.rb

```diff
@@ -0,0 +1,18 @@
+class DependenciesController < ApplicationController
+  include LatestVersion
+  include RequirementsVersion
+
+  def show
+    latest_version_by_slug
+    @dependencies = @latest_version.dependencies
+    @version_numbers = []
+
+    @version_numbers = @dependencies.collect do |dependency|
+      dep_resolver(
+        dependency.rubygem.name,
+        dependency["requirements"],
+        Version.numbers_from_gemid(dependency["rubygem_id"]).to_a.uniq
```

Comment:
Perhaps this method belongs in `Rubygems` (has multiple version number), where you could use `public_versions`. 
`uniq` or `to_a` seems unnecessary, why do we need them?

Reply:
I actually don't need that code. I can delete it. I used that because while fetching some had multiple versions with same name(different platform ruby and java) and number.

Reply:
Should I only fetch where platform is ruby?

Reply:
Question: If two versions have same number where one is ruby and other is java. If both satisfy the condition of requirement, which one should I choose?